src/middleware/prepareOutput.js

```diff
@@ -49,10 +49,20 @@ module.exports = function (options, excludedMap) {
       if (options.postProcess) {
         if (promise && typeof promise.then === 'function') {
           promise.then(() => {
-            options.postProcess(req, res)
+            options.postProcess(req, res);
+            if (options.restify) {
+              next();
```

Comment:
Codacy has a fix for the issue: [Replace `next();` with `··next()`](https://app.codacy.com/gh/florianholzapfel/express-restify-mongoose/pullRequest?prid=8423227)
```suggestion
                next()
```